samples/AspNetCore/ControllerSample/Controllers/SampleController.cs

```diff
@@ -80,5 +80,20 @@ public async Task<ActionResult<Account>> Withdraw(Transaction transaction, [From
             await state.SaveAsync();
             return state.Value;
         }
+
+        /// <summary>
+        /// Method for throwing a BadRequest exception which will cause Dapr sidecar to throw an RpcException
```

Comment:
> BadRequest exception which will cause Dapr sidecar to throw an RpcException

minor detail: there's no exception in this code - it's returning an HTTP 400 via the `BadRequestResult` type.

Reply:
fixed